src/integration-test/java/io/aiven/kafka/connect/http/IntegrationTest.java

```diff
@@ -175,17 +168,12 @@ void testBasicDelivery() throws ExecutionException, InterruptedException {
             sendFuture.get();
         }
 
-        TestUtils.waitForCondition(
-            () -> bodyRecorderHandler.recorderBodies().size() >= expectedBodies.size(),
-            15000,
-            "All requests received by HTTP server"
-        );
-        log.info("Recorded request bodies: {}", bodyRecorderHandler.recorderBodies());
```

Comment:
same question about logging the entries for debugging

Reply:
The new assertion will show in the fail message what was in this list. Unless the test fails I guess nobody reads the logs, so I don't think it's useful.